src/addons/rails-modal-and-panel/panel.scss

```diff
@@ -1,17 +1,16 @@
 .panel {
   --rm-panel-width: 40%;
 
-  height: 100vh;
-  width: var(--rm-panel-width);
-  z-index: var(--rm-z-index-drawer);
   position: absolute;
+  z-index: var(--rm-z-index-drawer);
   right: calc(-1 * var(--rm-panel-width)); // this pushes the panel off the sceen to the right
+  width: var(--rm-panel-width);
+  height: 100vh;
+  padding-bottom: var(--rm-space-large);
   background: var(--rm-color-background);
   color: var(--rm-color-on-background);
-  transition: right var(--rm-transition-panel);
-
   overflow-y: scroll;
-  padding-bottom: var(--rm-space-large);
+  transition: right var(--rm-transition-panel);
```

Comment:
@Jeremy-Walton Are we standardizing that all modals will transition in this way? Just curious on the context.

Reply:
This is the old panel code. I'm not changing behavior, just making what we already have compliant with formatting standards.

Rewriting this and the modal code is a completely separate task